src/chapters/tableau-part2/studio.rst

```diff
@@ -1,2 +1,226 @@
 Studio: Tableau Part 2 - Data Preparation
-=========================================
\ No newline at end of file
+=========================================
+
+Getting Started
+---------------
+
+Background:
+^^^^^^^^^^^
+
+This dataset used in this studio was created by the Institute of Museums and Library Services (IMLS).  We 
+will be exploring data from the Fiscal Year 2014.  Documentation of this survey can be found `here <https://www.imls.gov/sites/default/files/fy2014_pls_data_file_documentation.pdf>`__ if you would like to learn more about the survey.  The survey collected data for each state and the District of Columbia.
+
+Working Through the Studio:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+| You will be placed into one of eight groups.  The group number reflects the region you have been assigned in the table below.  As a team, one group member can be in charge of Tableau and screen share while other group members provide direction and suggestions.  At the end of studio, you will present your favorite chart or final dashboard to the class as a whole.
+
+| While this is a group assignment, for your own portfolio it is recommended that you make your own version of this studio.
+
+Setting Up the Studio:
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. _tableau-part2-setup-studio:
+
+Download `library data set <https://www.kaggle.com/imls/public-libraries?select=libraries.csv%C2%A0>`__. There are 2 csv files. To download all csv files at once, click on the "Download" bubble next to the "New Notebook" bubble.  An orange arrow is pointing to the "Download" bubble in the image below.
+
+.. figure:: figures/download-multi-files.png
+   :alt: An orange arrow points to the "Download" bubble, highlighting the Download option of kaggle.
+
+#. Open the csv file in Tableau Public.  
+
+   A. It is a larger file than we have previously used and may take a few minutes to open. 
+   #. `Use your downtime wisely <https://xkcd.com/303/>`__.
+
+#. When you open this data set, you will want to create a relationship between the *libraries.csv* file and the *states.csv* file using branch libraries.
+
+#. Final outcome of this studio will include in a single Tableau Public viz:
+
+   A. 9 worksheets
+
+      #. 10 if you do the bonus 
+      
+   #. And, 1 dashboard 
+
+#. :ref:`Part A<tableau-part2-partA>` invites you to briefly explore the data set.  You will explore all the regions of the US.
+
+#. In :ref:`Part B <tableau-part2-partB>`, using your assigned region, you will be asked to answer questions using the collection type of your choice: a set or group.  
+
+#. In :ref:`Part C <tableau-part2-partC>`,  you will select 2 (or more) of your favorite charts and create a dashboard that highlights results from your selected region.
```

Comment:
I think this list is a little confusing because some of these things are about setup and some are just general studio requirements. Maybe separate into two lists.